pkg/kubectl/cmd/delete.go

```diff
@@ -69,10 +69,14 @@ Examples:
 				NamespaceParam(cmdNamespace).DefaultNamespace().
 				FilenameParam(flags.Filenames...).
 				SelectorParam(cmdutil.GetFlagString(cmd, "selector")).
-				ResourceTypeOrNameArgs(args...).
+				SelectedResourceTypeOrNameArgs(args...).
 				Flatten().
 				Do()
-			checkErr(r.Err())
+
+			if r.Err() != nil {
```

Comment:
my understanding is that checkErr is calling glog.FatalDepth and it's not returning on cmd.out. This prevents unit test to verify the output of the command.